lib/rubocop/cop/rspec/variable_definition.rb

```diff
@@ -69,7 +69,7 @@ def string?(node)
         end
 
         def symbol?(node)
```

Comment:
Maybe we should inline `#string?` and `#symbol?` now? They are only used once each.